### PR TITLE
Release notes for v2.5.1

### PR DESCRIPTION
**Pull request by davidkline-ms**
This change updates the release notes for the 2.5.1 UPM release.

### CODE REVIEW

**Comment by davidkline-ms** [comment]
/.azp run mrtk_docs

File: Documentation/ReleaseNotes.md [hunk 1]
```diff
@@ -7,250 +7,24 @@
 
 ## What's new
```
Comment (keveleigh) [inline Documentation/ReleaseNotes.md:8]: Should we add the two PRs just merged?

File: Documentation/ReleaseNotes.md [hunk 2]
```diff
@@ -7,250 +7,24 @@
...
 
-### Unity Package Manager (UPM) support
+>[NOTE]
```
Comment (keveleigh) [inline Documentation/ReleaseNotes.md:10]: I think this has to be

```suggestion
> [!NOTE]
```

**Review by keveleigh — APPROVED** [review]

File: Documentation/ReleaseNotes.md (hunk 1 above)
Comment (davidkline-ms) [reply Documentation/ReleaseNotes.md:8]: they are small fixes, like the two sample updates. i think we are fine without them.

they _will_ show up in the changelog (via the link contained in that file)

**Pull request merged**